styles.css

```
@@ -0,0 +1,419 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  overflow-x: hidden;
+}
+
+body {
+  background: #f4f5f6;
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+.container_wrapper {
+  max-width: 1300px;
+  margin: auto;
+  background: #fff;
+}
+
+a {
+  text-decoration: none;
+  color: #161616;
+}
+
+.article-post a {
+  text-decoration: none;
+}
+
+.social li a {
+  font-size: 1.3rem;
+  padding-right: 1rem;
+  color: #666;
+}
+
+.footer-navigation li a {
+  padding: 0 0.8rem;
+  border-right: 1px solid #e5e5e5;
+  color: #666;
+  font-family: sans-serif;
+  font-size: 0.9rem;
+}
+
+.container_wrapper .nav-item a {
+  font-weight: 700;
+  color: #000;
+  font-size: 1.25rem;
+  margin-bottom: 0;
+  padding: 0 1rem;
```

Let's set this padding to 0.
```suggestion
  padding: 0;
```

Also, let's add style for the parent element, `li`

```css
.nav-item {
    flex-grow: 1;
    text-align: center;
}
```

And for the list `ul`

```css
ul.bd-highlight {
    width: 100%;
}
```